deployments/quick-start/setup.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/sh
+
+# Check script dependencies
+for command in http jq
+do
+    if ! hash $command 2>/dev/null; then
+        echo $command is not available, please install $command
+        exit 1
+    fi
+done
+
+# Inspect the deployment file deployments/quick-start/vault.yaml. The deployment starts Vault in development mode with \
+# the root token set to vault-root-token. It is also started using http. In production, you should run Vault over https.
+kubectl apply -f vault.yaml
+
+#2. Setup Vault
+# Wait 5 minutes for vault to be deployed.
+maxWaitSecond=300
+vaultPod=$(kubectl get pods -l app=vault | grep ^vault* |awk '{print $1}')
+while [ $maxWaitSecond -gt 0 ] && [ -z "$vaultPod" ]
+do
+    sleep 1
+    echo waited 1 second for kubernetes to deploy Vault
+    maxWaitSecond=$(($maxWaitSecond-1))
+    vaultPod=$(kubectl get pod | grep ^vault* |awk '{print $1}')
+done
+if [ -z "$vaultPod" ]
+then
+    exit 1
+fi
+
+echo "Vault pod name: $vaultPod"
+# Wait 10 seconds for vault to be running.
+maxWaitSecond=10
+vaultStatus=$(kubectl get pod "$vaultPod" -o=jsonpath='{.status.phase}')
+while [ $maxWaitSecond -gt 0 ] && [ "$vaultStatus" != "Running" ]
+do
+    sleep 1
+    echo waited 1 second for Vault up and running
+    maxWaitSecond=$(($maxWaitSecond-1))
+    vaultStatus=$(kubectl get pod "$vaultPod" -o=jsonpath='{.status.phase}')
+    echo Vault pod status "$vaultStatus"
+done
+if [ "$vaultStatus" != "Running" ]
+then
+    exit 1
+fi
+
+# 2.1 Port forward vault
+nohup kubectl port-forward $vaultPod 8200 &
+
+# Set environment variables and authenticate
+export VAULT_ADDR=http://127.0.0.1:8200
+vault auth vault-root-token
```

Comment:
I don't think this bit of functionality would be super useful as the script is probably only useful for standing up a test deployment. In a prod environment, you will most likely have different policies regarding the expiration and certificate issuance for your PKI backends. In addition, the Vault policies would highly likely be very different from the one shipped with the quick start deployment.